app/controllers/general.php

```diff
@@ -1028,16 +1019,36 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
         $response->text($content);
     });
 
-include_once __DIR__ . '/shared/api.php';
-include_once __DIR__ . '/shared/api/auth.php';
-
-App::wildcard()
+Http::wildcard()
     ->groups(['api'])
     ->label('scope', 'global')
     ->action(function () {
         throw new AppwriteException(AppwriteException::GENERAL_ROUTE_NOT_FOUND);
     });
 
 foreach (Config::getParam('services', []) as $service) {
```

Comment:
I think this foreach can be removed now that we manually include all the files below?